Update to nightly-2022-09-06 (#934)
EmbarkStudios/rust-gpu: CHANGELOG.md

### Changed 🛠️

- 🚨BREAKING🚨 Migrated from `register_attr` to `register_tool`. [More information](docs/src/migration-to-register-tool.md).
- Updated toolchain to `nightly-2022-09-06`
- Updated `glam` to `0.22`

> - Removed `glam::BVec` support (they are no longer `#[repl(simd)]` in `glam`, as Rust doesn't support vector of bool types)

Reviewer: ```suggestion
- Removed `glam::BVec` support (they are no longer `#[repl(simd)]` in `glam`, as Rust doesn't support SIMD vectors with `bool` elements)
```
